scene/3d/spatial.cpp

```diff
@@ -391,6 +404,20 @@ Vector3 Spatial::get_rotation() const{
 
 	return data.rotation;
 }
+
+Vector3 Spatial::get_rotation_deg() const {
+
+	return get_rotation() * 180.0 / Math_PI;
+}
+
+// Kept for compatibility after rename to set_rotd.
+// Could be removed after a couple releases.
+Vector3 Spatial::_get_rotation_deg() const {
+
+	WARN_PRINT("Deprecated method Spatial._get_rotation_deg(): This method was renamed to get_rotation_deg. Please adapt your code accordingly, as the old method will be obsoleted.");
```

Comment:
I guess nobody uses this method, and even if he does, he shouldn't expect it to work, since it is private API.